src/components/input/DurationInput.js

```diff
@@ -288,16 +244,34 @@ class DurationInput extends PureComponent {
     );
 
     const boxProps = pickBoxProps(others);
+    const seconds = this.state.seconds;
+
+    const secondsInMinute = 60;
+    const secondsInHour = secondsInMinute * 60;
+
+    const splitSecondsIntoHoursMinutesSeconds = seconds => {
```

Comment:
move this out of the render function and calculate the values once in render
now you're creating this function on every render and calling it 9 times per render.